internal/controller/driver_controller.go

```diff
@@ -223,6 +225,27 @@ func (r *DriverReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	return ctrl.Result{}, err
 }
 
+func (r *driverReconcile) hasSnapshotMetadataService() bool {
```

Comment:
The name SnapshotMetadataService can be read as "has SnapshotMetadata Service" which is confusing. Can we rename the function to `hasSnapshotMetadataServiceCR`?